app/workers/pubsubhubbub/confirmation_worker.rb

```diff
@@ -60,9 +60,7 @@ def confirm_callback
   end
 
   def callback_get_with_params
-    HTTP.headers(user_agent: 'Mastodon/PubSubHubbub')
-        .timeout(:per_operation, write: 20, connect: 20, read: 50)
-        .get(subscription.callback_url, params: callback_params)
+    Request.new(:get, subscription.callback_url, params: callback_params).perform
```

Comment:
I am unifying them in one place, which I think is the primary benefit of this. Also, letting a request stall for 90 seconds wasn't good behaviour, I think. We'd rather fail and retry later.